bedrock/products/templates/products/relay/faq.html

```diff
@@ -33,7 +33,7 @@ <h3> {{ ftl('faq-question-missing-emails-question-2') }}</h3>
           <ul class="mzp-u-list-styled">
             <li>{{ ftl('faq-question-missing-emails-answer-reason-spam') }}</li>
             <li>{{ ftl('faq-question-missing-emails-answer-reason-blocked-2') }}</li>
-            <li>{{ ftl('faq-question-missing-emails-answer-reason-size', size=settings.RELAY_EMAIL_SIZE_LIMIT_KB) }}</li>
+            <li>{{ ftl('faq-question-missing-emails-answer-reason-size', size=settings.RELAY_EMAIL_SIZE_LIMIT_MB) }}</li>
```

Comment:
**Nit:** `{size}` being displayed instead of number:
![Screenshot 2023-07-28 at 3 55 59 PM](https://github.com/mozilla/bedrock/assets/42309026/e746a73f-68fc-49a5-9523-adba2c5f72b3)